treeherder/etl/push.py

```diff
@@ -9,23 +9,22 @@
 logger = logging.getLogger(__name__)
 
 
-def store_push(repository, result_set):
-    result_set_revision = result_set.get('revision')
-    if not result_set.get('revision'):
+def store_push(repository, push_perm):
```

Comment:
maybe instead of ``push_perm`` it could be ``push_dict`` since this is a dict of push data coming in.